student.rb

```diff
@@ -1,6 +1,8 @@
 require './person'
 
 class Student < Person
+  attr_accessor :clasroom
+
```

Comment:
- [ ] Something similar is happening to your Student class, as you need to complete setting the relationship between the student and classroom class. So in this case the code is a bit different and may look something like this:
```
def classroom=(classroom)
  @classroom = classroom
  classroom.students.push(self) unless classroom.students.include?(self)
end
```
And again, this should set the proper relationship between students and the classroom. Take into account that both sides are necessary to set the relationship between classes.
You can take a look at [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) article if you need further information or reminder about associations and relationships.